crates/types/src/net/replicated_loglet.rs

```diff
@@ -29,6 +29,13 @@ define_rpc! {
     @response_target = TargetName::ReplicatedLogletAppended,
 }
 
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+pub enum SequencerStatus {
```

Comment:
Not sure if we will need to extend this list of SequencerStatus with more variants